content/guides/references/error-messages.md

```diff
@@ -428,23 +428,50 @@ it('does not forget to return a promise', () => {
 })
 ```
 
-### <Icon name="exclamation-triangle" color="red"></Icon> `cy.visit()` failed because you are attempting to visit a second unique domain
```

Comment:
@jennifer-shehane are we able to remove these errors since they don't exist in cypress 12 but exist in other versions of cypress? @emilyrohrbough and I were talking and think we might need to keep them since the on links in the App <12.0.0 would be broken

Reply:
on link PR https://github.com/cypress-io/cypress-services/pull/5040
app changes https://github.com/cypress-io/cypress/pull/24891

Reply:
@AtofStryker Yes, we keep these error messages forever so that users of old versions have a correct message to go to. 

I would suggest keeping the old hashtag the same so this isn't a dead link and updating the message of the content to mention that they should consider upgrading to v12 for x and x benefit. 

I'm sure we could have some strategy for removal eventually, but no strategy is outlined for removal of these messages from older versions today.

Reply:
OK works for me. I will add them back in 😄

Reply:
should be addressed in 2080412b9. Added a little `note` that explains that the error is only for v11 and under